.circleci/config.yml

```diff
@@ -0,0 +1,17 @@
+version: 2.0
+jobs:
+  build:
+    docker:
+      - image: circleci/elixir:1.10
```

Comment:
We've been using our own image for Circle builds. Probably not needed here, but I would use it for consistency.